examples/templates/suite_com/test_com.py

```diff
@@ -59,21 +59,50 @@ def test_run(self):
         logging.info(
             f"--------------- RUN: {self.test_suite_id}, {self.test_case_id} ---------------"
         )
-        # send 20 requests over the connected channel and check if the
-        # command was successfully sent
+
+        # send 20 requests (without context manager) over the connected channel
+        # and check if the command was successfully sent
         for _ in range(20):
             req = b"\x02\x04\x06"
             logging.info(f"send request {req} over {com_aux.name}")
             state = com_aux.send_message(req)
             logging.info(f"request excecution state: {state}")
             self.assertEqual(state, True)
 
+        # check messages have not been queued in as
+        # context manager wasn't used
+        self.assertTrue(com_aux.queue_out.empty())
+
+        # send 20 requests (with the context manager over the connected channel
+        #  and check if the command was successfully sent
+        with com_aux.collect_messages():
+            for _ in range(20):
+                req = b"\x02\x04\x06"
+                logging.info(f"send request {req} over {com_aux.name}")
+                state = com_aux.send_message(req)
+                logging.info(f"request excecution state: {state}")
+                self.assertEqual(state, True)
```

Comment:
You should call com_aux.receive_message() at least to better understand the feature